deck.py

```diff
@@ -0,0 +1,17 @@
+"""Deck class"""
+"""A Deck collaborates with Card and Shoe"""
+"""It is responsible for creating all 52 cards from the Card class
+   to be delivered to the Shoe"""
+from card import Card, ranks, suits
+
+
+class Deck:
+    """The deck will collect all 52 cards. It will be able to shuffle"""
+    def __init__(self):
+        self.deck = [Card(rank, suit)
+                     for rank in ranks
+                     for suit in suits]
+        deck = self.deck
+
+    def __str__(self):
+        pass
```

Comment:
What this does means that every time you try to print a Deck object -- given, you might not do that, but this is still important -- nothing at all happens. This is a bad idea.